.github/workflows/review.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: GPTcode-reviewer
-        uses: PierreGode/GPTcode-reviewer@dev
+        uses: PierreGode/GPTcode-reviewer@PR-LOG
```

Comment:
Using a branch name like 'PR-LOG' may lead to issues if the branch is deleted or renamed. Consider using a stable tag or version instead.